packages/x-components/cypress.json

```diff
@@ -1,6 +1,6 @@
 {
   "baseUrl": "http://localhost:8080",
-  "defaultCommandTimeout": 30000,
+  "defaultCommandTimeout": 7000,
```

Comment:
This is a silly question but...why is 7 seconds and not 8 or 6? Is guess is the minimum you found to pass the tests?

Reply:
Can we remove it and try with the default (4s)?

Reply:
With the default values, and different configurations (ci, throttled 4x, throttled 6x) I've  only seen it fail on the search-box test. I would merge this as it is. It is a big improvement. In case it fails  in the e2e tests we only have to wait 14 seconds, not 1 minute  and a half like before.